### DISCUSSION TITLE
MapRotation.Repeat

### DISCUSSION THREAD

**Discussion by ZhenyaMartynov** [Sokol-Games-GmbH/PixelCombats.PublicApi#167] [Ideas]
**добавить свойство repeat к Map.MapRotation**.

Свойство будет отвечать за то, будут ли повторятся карты из списка, пока не прошли другие.

Например если оно true, то карты будут повторяться, например 2 раза 1 и та же карта будет выбиратся.

Иначе же пока не будет окончен цикл всех карт, они будут идти поочередна из списка карт.

**Comment by kkohno** [comment]
обдумаем

**Comment by kkohno** [comment]
скоро будем заниматься всеми этими переключениями, Вашу идею учтем

**Reply by SobakaBarabaka74** [reply]
Понял))
